### DISCUSSION TITLE
Wacom Tablet drivers

### DISCUSSION THREAD

**Discussion by mrtsb3061** [community/community#63099] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi I see some great work on Wacom Tablet drivers, question is they're a driver for the Intuos Pro s or M as this tablet seems to exhibit problems with Monterey. Thanks for any feedback. Cheers Mike

**Comment by w8ste** [comment]
@mrtsb3061 Hi, i found a driver for the Intuos Pro S on the official Wacom website [here](https://www.wacom.com/en-us/support/product-support/drivers)
I hope this helps you :)

**Comment by queenofcorgis** [comment]
Hi @mrtsb3061 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this like @w8ste linked above. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
